lib/window.js

```diff
@@ -19,7 +19,7 @@ module.exports = function Window (config, path, opts) {
 
       var config = ${JSON.stringify(config)}
       var data = ${JSON.stringify(opts.data)}
-      var title = ${JSON.stringify(opts.title || 'Patchwork')}
+      var title = ${JSON.stringify(opts.title || "Patchwork" )}
```

Comment:
Is there a specific reason for switching to use double quotes on some strings?

Patchwork code mostly conforms to [JavaScript Standard Style](https://standardjs.com/rules.html) (and I personally use this as a linter when working on it), and this only allows double quotes (") when you want to escape single ones.